app.rb

```diff
@@ -72,7 +73,8 @@
         begin
           issue.save!
           actions['redmine'] = true
-        rescue RestClient::UnprocessableEntity => e
+        rescue RestClient::UnprocessableEntity => exception
+          Raven.capture_exception(exception)
           puts "Failed to save issue #{issue} for PR #{pull_request}: #{e.message}"
```

Comment:
make sure you update e.message to exception.message

Reply:
Good catch!